### DISCUSSION TITLE
Connecting and disconnect

### DISCUSSION THREAD

**Discussion by ewookscorp** [opencardev/crankshaft#459] [General]
Hi I just installed this dash version to in my RPi 4. When connecting Android device phone detect and trying to connect but it wont connect.  Phone keep asking "Try again". How get this work also with this Dash? Early I tried with Crankshaft image and that worket fine but I like try develop my own plugin for this Dash.

**Comment by Geekyadz** [comment]
During my testing today I had this with my Motorola Moto Z2 Play; I ended up unplugging it from the Pi then reconnecting it and AA kicked in
